packages/plugin-ext/src/common/editor-options.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/**
+ * The style in which the editor's cursor should be rendered.
+ */
+export enum TextEditorCursorStyle {
```

Comment:
This looks like it's copied from vscode. If so, please attribute with MIT license in the header.